packages/netlify-cms-core/src/constants/configSchema.js

```diff
@@ -169,7 +177,7 @@ class ConfigError extends Error {
  * the config that is passed in.
  */
 export function validateConfig(config) {
-  const ajv = new AJV({ allErrors: true });
+  const ajv = new AJV({ allErrors: true, jsonPointers: true });
```

Comment:
`ajv-errors` sets this to true anyways and will log a warning if not set: https://github.com/epoberezkin/ajv-errors/blob/4e8cc0ed941b333b97a265ec599a0f62df625108/index.js#L5